preload/data/ui/chart-editor/dialogs/user-guide.xml

```diff
@@ -1,10 +1,407 @@
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/chords.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Chords\ndoubles, triples, or quads."/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/jumpstream.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Jumpstream."/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/handstream.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Handstream."/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/quadstream.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Quadstream."/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/jumptrill.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Jumptrill"/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/split-jumptrill.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Split jumptrill [13] [24]"/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/split-jumptrill-2.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Split jumptrill [14] [23]"/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/grace-notes.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Grace notes\nSlightly offset notes that is featured in Erect Remixes frequently.\nCould be mistaken for a chord at first glance due to how close they are in-game."/>
+							</vbox>
+
+							<vbox id="pattern">
+								<image resource="images/patterns/burst.png" imageScale="0.5"/>
+								<label id="img-desc" width="100%" text="Burst\nA sudden burst of notes, requires very good note placements and playtesting."/>
+							</vbox>
+
+						</grid>
+					</scrollview>
+				</vbox>
+					<vbox width="100%" style="padding-top:10px;">
+						<label width="100%" text="references\n Etterna's help screen. \n https://www.flashflashrevolution.com/vbz/showthread.php?t=127628 \n https://docs.google.com/document/u/0/d/1Yb9hiewZFs27F3mB6V3reiRBA3E00zb_xfjGVcsnYhg/mobilebasic#h.tv62oe7zl3qz \n https://sectofmysticwisdom.com/" style="font-size: 12px; text-color: #808080" />
+					</vbox>
+				</scrollview>
+			</vbox>
+		</box>
+
+		<box width="100%" text="Tips">
+			<vbox width="100%">
+				<scrollview width="950" height="500" contentWidth="100%" scrollMode="default">
+					<vbox width="100%">
+						<section-header text="Tips" id="heading" />
+					</vbox>
+
+					<vbox width="100%">
+						<label width="100%" text="Treat these as tips I recommend to follow and not rules you should follow as charters have different perspectives on what is considered a bad chart and a good chart.\n\n\n" style="font-weight: bold" />
```

Comment:
We'll probably have our charting team proofread and tweak these tips to help match their style of charting.